Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.5 AS buildenv
+FROM golang:1.13.5 AS buildenv
```

Comment:
Can you change to 1.13 so the latest 1.13.* is pulled ?

Reply:
Sure thing.